code/game/Q3_Interface.cpp

```diff
@@ -26,6 +26,7 @@ along with this program; if not, see <http://www.gnu.org/licenses/>.
 //	is not directly portable from engine to engine.
 //
 //	-- jweier
+#include <array>
```

Comment:
I'm not entirely sure about the right place to include <array> according to q3 conventions, so I used normal c++ best practices and only included it in the .cpp that actually needs it